homeassistant/components/light/tplink.py

```diff
@@ -42,12 +42,16 @@ def brightness_from_percentage(percent):
     return (percent*255.0)/100.0
 
 
+# Travis-CI runs too old astroid https://github.com/PyCQA/pylint/issues/1212
```

Comment:
Travis is not running an outdated version, it is running the version that we specified in our requirements: https://github.com/home-assistant/home-assistant/blob/dev/requirements_test.txt#L5